.github/workflows/windows-vcpkg.yml

```diff
@@ -27,11 +27,6 @@ jobs:
       
     steps:
     - uses: actions/checkout@v2    
-
```

Comment:
Question : on **ubuntu**, we keep asking for python 3.8. Does this mean that no default python is installed ?

Reply:
Python 3.8.10 is installed on ubuntu-latest (20.04), see [here](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#language-and-runtime).